lib/net/common.js

```diff
@@ -187,7 +187,7 @@ exports.ZERO_NONCE = Buffer.alloc(8, 0x00);
  * @default
  */
 
-exports.MAX_INV = 50000;
```

Comment:
even though they changed MAX_MESSAGE size they have not changed MAX_INV number.
So 50k remains.